src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,60 @@
+import { FC, memo } from 'react';
+import cn from 'classnames';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+interface Props {
+  people: Person[],
+  selectedPersonSlug: string,
+}
+
+export const PeopleTable:FC<Props> = memo(({ people, selectedPersonSlug }) => {
+  return (
+    <table
+      data-cy="peopleTable"
+      // eslint-disable-next-line max-len
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
```

Comment:
Let's move it so that we don't add a comment above
```suggestion
      className="
      	table
      	is-striped
      	is-hoverable
      	is-narrow
      	is-fullwidth
      "
```

Reply:
fix it